scripts/release/release.js

```diff
@@ -0,0 +1,52 @@
+#! /usr/bin/env node
+/* eslint-disable no-console */
+// eslint-disable-next-line import/no-extraneous-dependencies
+const pacote = require('pacote');
+const { execSync } = require('child_process');
+const { exit } = require('process');
+const packageJson = require('../../package.json');
+
+// If on travis setup git to be able to push the tags
+const setupGit = () => {
```

Comment:
Should this thrown an error if not in Travis ENV? or assuming this is supporting locale releases, should this run the prepublishOnly script, `npm whoami && check-installed-dependencies.

I'm wondering if this should be a promise to ensure git setup is correct / validate before running publish

Reply:
if the git setup fails at any point it will exit the script since it's wrapped in the async `release` function. I'm also fine with (people who have privs) still having the option to release locally if there is some type of emergency situation.